_examples/idv/handlers.session.go

```diff
@@ -208,3 +212,83 @@ func showPrivacyPolicyPage(c *gin.Context) {
 	render(c, gin.H{}, "privacy.html")
 	return
 }
+
+/* For Testing */
+
+func showDigitalPage(c *gin.Context) {
```

Comment:
```suggestion
func showDigitalIdPage(c *gin.Context) {
```

Reply:
That was for local testing and had been removed.